openmetadata-ui/src/main/resources/ui/src/pages/PipelineDetails/PipelineDetailsPage.component.tsx

```diff
@@ -514,6 +515,34 @@ const PipelineDetailsPage = () => {
       });
   };
 
+  const deletePostHandler = (threadId: string, postId: string) => {
```

Comment:
Can we avoid duplicating this deletePostHandler logic?

Reply:
@vivekratnavel,  We will need `deletePostHandler` in every page component, because we need to maintain internal state of that page component.

Reply:
We can write a utility method and pass a method to set state just like how we pass "onConfirmation" to child components. The signature of the utility method will be something like 
```
const deletePostHandler = (threadId: string, postId: string, onEntityThreadUpdate: )
```

Reply:
@Sachin-chaurasiya Can you please take care of this refactoring?

Reply:
@vivekratnavel , I already did, move out the API call to util method and calling the same method from deletePostHandler.